### DISCUSSION TITLE
Pokemon moves

### DISCUSSION THREAD

**Discussion by nasserkessas** [Pokemon-PythonRed/Pokemon-PythonRed#76] [General]
Hello,

I was wondering whether Pokémon moves are going to be implemented into the game in the future. I played the game and found it a bit weird that the Pokémon don't have moves as a former Pokémon Omega Ruby player.
I think it shouldn't be hard to include the moves in a `.json` file similar to the Pokémon. I would be happy to help do this if you choose to.

I had a look at this project and really liked the idea and would hopefully like to contribute in the future.

Thanks

**Comment by riverpaper** [comment]
Thank you for your review!

I agree that not having Moves is a little weird. The problem is that I don't know how to go about implementing them. If you have any ideas or are able to do it yourself, that would be greatly appreciated.

**Comment by nasserkessas** [comment]
No problem, I have done all this and created a [PR](https://github.com/Pokemon-PythonRed/Pokemon-PythonRed/pull/83).
I'm looking forward to your review!
